server/server.go

```diff
@@ -29,6 +29,19 @@ func newCribbageServer(dbFactory persistence.DBFactory) *cribbageServer {
 func (cs *cribbageServer) NewRouter() http.Handler {
 	router := gin.Default()
 
+	router.LoadHTMLGlob("templates/*")
+	router.Static("/assets", "./assets")
+
+	router.GET("/", handleIndex)
+
+	// Simple group: user. Used for serving pages affiliated with a given user
+	user := router.Group("/user")
+	{
+		user.GET("/", handleGetUser)
+		user.GET("/:username", cs.handleGetUsername)
+		user.GET("/:username/game/:gameID", cs.handleGetUsernameGame)
+	}
```

Comment:
Let's put all of these routes in a new group `/wasm` so that we can use these same paths for the react frontend and just have the templates served up on the wasm html calls

Reply:
Maybe we should also group our api endpoints into a group to distinguish them from the wasm stuff?

Reply:
I think the "api" stuff, isn't so much "API", but just REST endpoints to hit. The wasm stuff is specific to wasm because our gowasm code knows how to hit it.